core/frontend/src/IModelConnection.ts

```diff
@@ -612,6 +617,26 @@ export abstract class IModelConnection extends IModel {
     }
     return ("number" === typeof this._projectCenterAltitude) ? this._projectCenterAltitude : undefined;
   }
+
+  /**
+   * Gets the context that allows accessing the metadata (see `@itwin/ecschema-metadata` package) of this iModel.
+   * The context is created lazily when this property is accessed for the first time, with an `ECSchemaRpcLocater` registered as a fallback locater, enabling users to register their own locater that'd take more priority.
+   * This means to correctly access schema context, client-side applications must register `ECSchemaRpcInterface` following instructions for [RPC configuration]($docs/learning/rpcinterface/#client-side-configuration).
+   * Server-side applications would also [configure RPC]($docs/learning/rpcinterface/#server-side-configuration) as needed.
+   *
+   * @note While a `BlankConnection` returns a valid `schemaContext`, it has an invalid locater registered by default, and will throw an error when trying to call it's methods.
+   * @beta
```

Comment:
@aruniverse do we want to switch from @beta to @public @preview ?

Reply:
no we can leave as is for now